ffi/input.lua

```diff
@@ -6,10 +6,6 @@ if util.isSDL() then
     end
 elseif util.isAndroid() then
     return require("ffi/input_android")
-    -- FIXME: Currently we support both ffi poll mode, and InkViewMain thread wrapper.
-    -- Nuke this global check once we switch the client to poll mode exclusively.
-elseif util.isPocketbook() and _G.POCKETBOOK_FFI then
+elseif util.isPocketbook() then
     return require("ffi/input_pocketbook")
-else
-    return require("libs/libkoreader-input")
```

Comment:
@NiLuJe This else fallback is a bit weird. Just to be sure. No other devices need the libkoreader-input right?

Reply:
D'oh, I didn't realize it went on under the isPocketbook. Maybe that'll teach me to review things on my phone. :laughing:

Reply:
Put it back :)